stdlib/public/SDK/Foundation/Calendar.swift

```diff
@@ -109,8 +98,9 @@ public struct Calendar : Hashable, Equatable, ReferenceConvertible, _MutableBoxi
     ///
     /// - parameter identifier: The kind of calendar to use.
     public init(identifier: Identifier) {
-        let result = __NSCalendarInit(Calendar._toNSCalendarIdentifier(identifier).rawValue as NSString)!
-        _handle = _MutableHandle(adoptingReference: result)
+        // For some reason the shims require the parameter here to be an NSObject and not a NSCalendarIdentifier
```

Comment:
That was a comment from when it was previously giving me some strange errors due to build cruft; I can try again